### PR TITLE
Run with sql auth

### PR DESCRIPTION
**Pull request by JosephDSchwartz**
# Summary of PR changes

Allowing the use of SQL Auth instead of strictly domain integrated security. This will open up the ability to run the action on more than just Windows runners.

## PR Requirements
- [X] For major, minor, or breaking changes, at least one of the commit messages contains the appropriate `+semver:` keywords.
  - See the *Incrementing the Version* section of the repository's README.md for more details.
- [X] The examples in the repository's `README.md` have been updated with the new version.  
  - See the *Incrementing the Version* section of the repository's README.md for more details on how the version will be incremented.
- [X] The action code does not contain sensitive information.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,19 +4,24 @@ A GitHub Action that takes in a list of dependency scripts for a database, downl
...
+| `db-name`                 | true        | N/A     | The name of the database where the dependency files will run.                                                                                            |
+| `dependency-list`         | true        | N/A     | A json string containing a list of objects with the name of the dependency package, the version, and the url where the package is stored.                |
+| `use-integrated-security` | true        | false   | Use domain integrated security. If false, a db-username and db-password should be specified. If true, those parameters will be ignored if specified.     |
+| `username`                | true        | N/A     | The username to use to login to the database. This is required if use-integrated-security is false, otherwise it's optional and will be ignored.         |
+| `password`                | true        | N/A     | The password for the user logging in to the database. This is required if use-integrated-security is false, otherwise it's optional and will be ignored. |
```
Comment (hpractv) [inline README.md:25]: It would be nice if they way these arguments were specified (default & required) were uniform with the other similar actions.

Comment (JosephDSchwartz) [reply README.md:25]: I'm not sure what you mean. I thought this was the way I'd done them on other actions, but maybe I'm wrong. I do tend to change the way I do things from time to time. Do you mind elaborating or linking an action that is doing it the right way?

Comment (hpractv) [reply README.md:25]: Sure.  In this action: [tsqlt-test-runner #inputs](https://github.com/im-open/tsqlt-test-runner#inputs), the user name and pwd are named slightly different and not required.

Comment (JosephDSchwartz) [reply README.md:25]: Ah I see what you're getting at. Yeah you're right. I'll update those things.

Comment (JosephDSchwartz) [reply README.md]: Done! I did it a bit fast, so let me know if I screwed anything up!

**Review by hpractv — APPROVED** [review]
👍🏽

**Pull request merged**